.travis.yml

```diff
@@ -108,6 +118,7 @@ before_script:
   # urdfdom
   - git clone https://github.com/ros/urdfdom
   - cd urdfdom
+  - git checkout 06f5f9bc34f09b530d9f3743cb0516934625da54
```

Comment:
A small comment about why this commit would go a long way. Thanks.

to quote..
"To fix compilation, we will switch the dependencies to use the commits used in the latest successful model generation"

Reply:
It is before, when we switch the commit of `console_bridge`.

Reply:
Ah ok!